conflux-sponsor-faucet-sdk/faucet-frontend/faucet-frontend.js

```diff
@@ -10,46 +11,87 @@ class Faucet {
      */
     constructor(url, address) {
         this.cfx = new Conflux({url: url});
-        this.proxy = this.cfx.Contract({
-            abi: faucetABI.abi,
+        this.faucet = this.cfx.Contract({
+            abi: faucet_contract.abi,
             address: address,
         });
+        this.cpc = this.cfx.Contract({
+            abi: cpc_contract.abi,
+            address: "0x0888000000000000000000000000000000000001",
+        });
     }
 
-    /**
-     * @dev apply to be sponsored
-     * @param dapp The address of dapp 
-     * @param address The sender address of apply message
-     */
-    async apply(dapp, address) {
-        let nonce = Number(await this.cfx.getNextNonce(address));
-        let estimateData = await this.proxy.applyFor(dapp).estimateGasAndCollateral();
-        let gas = new BigNumber(estimateData.gasUsed)
+    async tryWrapTx(call_func, params) {
```

Comment:
why comment removed?

every API in SDK should have comment that good/easy to understand,  since we will open source this sdk.

Reply:
comments added